test/model/adapters/sql/command_test.rb

```diff
@@ -0,0 +1,75 @@
+require 'test_helper'
+
+describe Lotus::Model::Adapters::Sql::Command do
+  let(:collection) { Object.new }
+  let(:query) do
+    query = Object.new
+    query.instance_variable_set(:@collection, collection)
+    query.define_singleton_method(:scoped, -> { @collection })
+    query
+  end
+
+  before do
+    @command = Lotus::Model::Adapters::Sql::Command.new(query)
+  end
+
+  describe '#create' do
+    describe 'when a Sequel::DatabaseError is raised' do
+      it 'raises a lotus error' do
+        collection.define_singleton_method(:insert) do |_|
+          raise Sequel::DatabaseError.new
+        end
+        -> { @command.create(Object.new) }.must_raise(Lotus::Model::Error)
```

Comment:
@Moratorius Hey, this looks good :+1: Extra bonus points if we assert the message:

```ruby
exception = -> { @command.create(Object.new) }.must_raise(Lotus::Model::Error)
exception.message.must_equal '...'
```

What do you think?